README.md

```diff
@@ -1,3 +1,18 @@
+# libcangjie
```

Comment:
I had actually never put that title in the README because I prefer it that way. 😛 I don't feel particularly strongly about it though, so I'd merge it in.